docs/tutorials/nb_shap_feature_elimination.ipynb

```diff
@@ -204,41 +200,49 @@
     "    'num_leaves': [3, 5, 7, 10],\n",
     "}\n",
     "search = RandomizedSearchCV(clf, param_grid)\n"
-   ]
+   ],
+   "outputs": [],
+   "metadata": {
+    "scrolled": false
+   }
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
    "source": [
     "## Apply ShapRFECV\n",
     "\n",
     "Now let's apply the [ShapRFECV](https://ing-bank.github.io/probatus/api/feature_elimination.html)."
-   ]
+   ],
+   "metadata": {}
   },
   {
    "cell_type": "code",
-   "execution_count": 4,
-   "metadata": {},
-   "outputs": [],
+   "execution_count": 5,
    "source": [
     "shap_elimination = ShapRFECV(\n",
     "    clf=search, step=0.2, cv=10, scoring='roc_auc', n_jobs=3)\n",
     "report = shap_elimination.fit_compute(X, y)"
-   ]
+   ],
+   "outputs": [],
+   "metadata": {}
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
    "source": [
     "At the end of the process, you can investigate the results for each iteration."
-   ]
+   ],
+   "metadata": {}
   },
   {
    "cell_type": "code",
-   "execution_count": 5,
-   "metadata": {},
+   "execution_count": 6,
+   "source": [
+    "#First 5 rows of first 5 columns\n",
+    "report[['num_features', 'features_set', 'val_metric_mean']]"
```

Comment:
Here you can also add the mean abs shap values.

Reply:
As i am thinking about this now, let's keep this part as it is, but somewhere later in the notebook, add a cell showing feature importances for the final feature set :)